keyboards/keebmaker/perigee/readme.md

```diff
@@ -0,0 +1,19 @@
+# Perigee
+
+![Perigee]
```

Comment:
There should be an image/photo here representative of the board.

```
![Perigee](https://imgur.com/someURL.jpg)
```

Reply:
All concerns have been addressed